src/app/games/game-list/game-list.component.scss

```diff
@@ -37,6 +37,26 @@ $-games-per-page: 5;
     align-items: center;
   }
 
+  &__score {
+  display: block;
+  width: 71.5px;
+  color: white;
+  &--red {
+    @include surface.fill-color(theme.$team-red);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 50%;
+  }
+  &--blu {
+    @include surface.fill-color(theme.$tem-blu);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 50%;
```

Comment:
All these props (display, justify-content, align-items, height) can be moved to a common class